generator/snowflake.go

```diff
@@ -152,7 +147,7 @@ func (r *Snowflake) ConfigureDestination() error {
 
 	// create stage
 	stageName := "perfstage" + r.generatorIdentifier
-	createStageQuery := "create stage " + stageName + " url='s3://" + r.stageS3BucketName + "' credentials = (AWS_KEY_ID = '" + credValue.AccessKeyID + "' AWS_SECRET_KEY = '" + credValue.SecretAccessKey + "' );"
+	createStageQuery := "create stage " + stageName + " url='s3://" + r.stageS3BucketName + "' credentials = (AWS_KEY_ID = '" + creds.AccessKeyID + "' AWS_SECRET_KEY = '" + creds.SecretAccessKey + "' );"
```

Comment:
curious - since we are concatenating strings with secrets here, is this a good practice?

Reply:
I don't know why we're doing this, is it a requirement to have them in the stage?

Reply:
yes - snowflake takes in the credentials as part of this command https://docs.snowflake.com/en/sql-reference/sql/create-stage.html